src/components/Menu/Menu.stories.tsx

```diff
@@ -20,9 +21,9 @@ export default {
 } as Meta<MenuProps>;
 
 export const Default: StoryObj<MenuProps> = {
-  render: (args) => (
```

Comment:
Converted these to `children` in `args` since it's no longer mulitiple React Elements